owncloudApp/src/main/res/values/strings.xml

```diff
@@ -754,7 +754,7 @@
     <!-- DeepLinks -->
     <string name="invalid_deep_link_format">Invalid link format</string>
 
-    <string name="fab_refresh_text">New content</string>
+    <string name="fab_refresh_text">Refresh </string>
```

Comment:
Good, but remove the space at the end, because it will appear as well in the label and can even give a wrong alignment impression of the text with the button